web/routes.go

```diff
@@ -39,6 +40,12 @@ var routes = []route{
 		"/health",
 		healthHandler,
 	},
+	route{
+		"job_status",
```

Comment:
kind of like rachel's way more, and it's only different in one place, so I will change. Thanks ben!